### ISSUE TITLE
Affected file does not get rebuilt when touching its dependency.

### ISSUE THREAD

**Issue by kittaakos** [crapo/sadlos2#199]
`/p/A.sadl`
```
uri "http://sadl.org/A.sadl".
import "http://sadl.org/B.sadl".
AA is a class.
```

`/p/B.sadl`
```
uri "http://sadl.org/B.sadl".
import "http://sadl.org/A.sadl".
MyB is an AA.
```

When commenting out the second line in `A.sadl`, `B.sadl` remains invalid.
Expected: should be valid.
Touching and saving `B.sadl` fixes the issue.

**Comment by kittaakos** [comment]
Here is another example:
`A.sadl`
```
uri "http://sadl.org/A.sadl".
// AAA is a class.
```

`B.sadl`
```
uri "http://sadl.org/B.sadl".
import "http://sadl.org/A.sadl".
```

`C.sadl`
```
uri "http://sadl.org/C.sadl".
import "http://sadl.org/B.sadl".
MyAAA is a AAA.
```

By default `C` has an error which is expected and correct behavior. By uncommenting the second line in `A` the error should gone in `C`. That is not happening.

**Comment by kittaakos** [comment]
PR: https://github.com/crapo/sadlos2/pull/204

**Comment by kittaakos** [comment]
Merged via #205. Closing issue.
